src/main/java/com/contrastsecurity/sdk/scan/ScanException.java

```diff
@@ -0,0 +1,23 @@
+package com.contrastsecurity.sdk.scan;
+
+import java.util.Objects;
+
+/** Indicates a condition that prevents Contrast Scan from completing an analysis */
+public class ScanException extends RuntimeException {
```

Comment:
Why does this not extend ContrastException instead?

Reply:
It should. Copy / paste error. I'll fix it